Gemfile

```diff
@@ -50,6 +50,7 @@ group :development, :test do
   gem 'selenium-webdriver'
   gem "factory_bot_rails", "~> 4.0"
   gem 'database_cleaner'
+  gem 'capistrano-sidekiq' 
```

Comment:
I think it should work having this gem under the `development` group only rather then `test` as well.
So you could try to move it there.